app/gallery/serializers/__init__.py

```diff
@@ -0,0 +1 @@
+from app.gallery.serializers.picture import PictureSerializer
```

Comment:
Vi har gått bort fra å bruke `__init__.py`-filer for å importere fra mapper og bruker heller absolutt import. Du kan derfor fjerne alle disse filene og skrie ut hele veien til klassen du vil importere ved importering 😊

Reply:
Da er disse filene slettet og direkte importering er lagt til